src/libsyntax_ext/deriving/clone.rs

```diff
@@ -115,7 +115,7 @@ fn cs_clone_shallow(name: &str,
         // set the expn ID so we can use the unstable struct.
         let span = span.with_ctxt(cx.backtrace());
         let assert_path = cx.path_all(span, true,
-                                        cx.std_path(&["clone", helper_name]),
+                                        cx.std_path(&[sym::clone, Symbol::intern(helper_name)]),
```

Comment:
I deliberately chose not to. I'm trying to pre-intern symbols that profiling shows are hot, rather than just any and every symbol that could be pre-interned. Because if we have enough pre-interned symbols the startup cost might become non-negligible.